itou/approvals/models.py

```diff
@@ -565,18 +565,26 @@ def notify_pole_emploi(self, at=None):
 
         pe_client = pole_emploi_api_client()
 
-        try:
-            encrypted_nir = pe_client.recherche_individu_certifie(
-                self.user.first_name, self.user.last_name, self.user.birthdate, self.user.nir
-            )
-        except PoleEmploiAPIException:
-            logger.info("! notify_pole_emploi approval=%s got a recoverable error in recherche_individu", self)
-            self.pe_save_should_retry(at)
-            return
-        except PoleEmploiAPIBadResponse as exc:
-            logger.info("! notify_pole_emploi approval=%s got an unrecoverable error in recherche_individu", self)
-            self.pe_save_error(api_enums.PEApiEndpoint.RECHERCHE_INDIVIDU, exc.response_code, at)
-            return
+        if self.user.jobseeker_profile and self.user.jobseeker_profile.id_national_pe:
```

Comment:
A priori maintenant on peut faire directement
```suggestion
        if self.user.jobseeker_profile.id_national_pe:
```
(et pareil dans la suite)

Reply:
Oui !